x-pack/solutions/security/plugins/security_solution/public/common/utils/timeline/use_show_timeline.test.tsx

```diff
@@ -120,41 +104,40 @@ it('shows timeline for users with timeline read access', async () => {
     timelinePrivileges: { read: true },
   });
 
-  const { result } = renderHook(() => useShowTimeline());
+  const { result } = renderUseShowTimeline();
   const showTimeline = result.current;
   expect(showTimeline).toEqual([true]);
 });
 
 describe('sourcererDataView', () => {
   it('should show timeline when indices exist', () => {
     mockUseSourcererDataView.mockReturnValueOnce({ indicesExist: true, dataViewId: 'test' });
-    const { result } = renderHook(() => useShowTimeline());
+    const { result } = renderUseShowTimeline();
     expect(result.current).toEqual([true]);
   });
 
   it('should show timeline when dataViewId is null', () => {
```

Comment:
Per the last comment, if you can add a comment for this and the below test why if you don't have indices and no dataview you would still want to show the timeline?